client/view/index/Index.html

```diff
@@ -135,5 +209,44 @@ <h2>Try an example deck</h2>
 	components: {
 		SpreadsheetInput,
 	},
-}
+	methods: {
+		prepareDeleteSeenWorkbook: function(element) {
+			var sure = document.querySelector(".are-you-sure-window");
+			sure.style.display = "block";
+			sure.style.opacity = '1';
+
+			var key = element.getAttribute("data-key");
+			var name = element.getAttribute("data-name");
+			var remove = document.querySelector(".are-you-sure-window .remove");
+			document.querySelector(".are-you-sure-window .here").innerHTML = name;
```

Comment:
In this case, instead of manipulating the DOM directly, you can make the template dynamic:

```html
<div class="are-you-sure-window">
	<div>
		<h2>Are you sure?</h2>
		<p>Are you sure you want to remove the {deckBeingDeleted} deck from this app?</p>
	</div>
</div>
```

and then update the component state with a

```js
this.set({
	deckBeingDeleted: name
})
```
call in the method.